webui/react/src/utils/types.ts

```diff
@@ -156,7 +166,7 @@ export const oneOfProperties = <T>(obj: any, props: string[]): T => {
 };
 
 // size in bytes
-export const checkpointSize = (checkpoint: Checkpoint): number => {
+export const checkpointSize = (checkpoint: Checkpoint | CheckpointWorkload): number => {
```

Comment:
**suggestion:** what do you think about moving trial specific type utilities into `utils/trial`? `utils/types` is starting to get a bit large

Reply:
we can leave that work to be a separate ticket. Basic idea of breaking up `utils/types` into the respective type it is dealing with.

Reply:
in general that sounds good. this specific one is also used in the checkpoint modal module and isn't specific to trials.

Reply:
https://github.com/determined-ai/determined/pull/1674/commits/f34d5ce2bf2dd94c95872adba1ab03919cdc284a